### PR TITLE
App strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: atox/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -145,11 +145,11 @@
     <string name="in_call_with">In call with %1$s</string>
     <string name="tox_id_error_self_add">You can\'t add yourself</string>
     <string name="save">Save</string>
-    <string name="pref_nospam">Nospam</string>
-    <string name="pref_nospam_description">Changing your nospam means that your Tox ID changes, and that your current Tox ID can no longer be used to send you friend requests</string>
+    <string name="pref_nospam">NoSpam</string>
+    <string name="pref_nospam_description">Changing your NoSpam changes your Tox ID, meaning friend requests must be sent to the updated one.</string>
```
Comment (comradekingu) [inline atox/src/main/res/values/strings.xml:149]:
```suggestion
    <string name="pref_nospam_description">Changing your NoSpam alters your Tox ID, meaning friend requests must be sent to the updated one.</string>
```

File: atox/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -109,7 +109,7 @@
     <string name="from">From</string>
     <!-- Status dialog -->
     <string name="status_dialog_edit_user_status">Edit your status</string>
-    <string name="mimetype_handler_not_found">Unable to find something that can handle that media type %1$s</string>
+    <string name="mimetype_handler_not_found">No app installed to handle that media type %1$s</string>
```
Comment (comradekingu) [inline atox/src/main/res/values/strings.xml:112]:
```suggestion
    <string name="mimetype_handler_not_found">No app installed to handle the \"%1$s\" media type</string>
```
?

Comment (robinlinden) [reply atox/src/main/res/values/strings.xml:112]: Yes, please!

File: atox/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -122,20 +122,20 @@
     <string name="pref_heading_address">Address</string>
     <string name="pref_heading_port">Port</string>
     <string name="pref_heading_proxy">Proxy</string>
-    <string name="warn_proxy_broken">Bad proxy configured. Fix or disable before using app.</string>
+    <string name="warn_proxy_broken">Set up your proxy correctly (or turn it off) before using the app.</string>
```
Comment (robinlinden) [inline atox/src/main/res/values/strings.xml:125]: Can you remove the full stop now that this is a single sentence?

File: atox/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -145,11 +145,11 @@
     <string name="in_call_with">In call with %1$s</string>
     <string name="tox_id_error_self_add">You can\'t add yourself</string>
     <string name="save">Save</string>
-    <string name="pref_nospam">Nospam</string>
-    <string name="pref_nospam_description">Changing your nospam means that your Tox ID changes, and that your current Tox ID can no longer be used to send you friend requests</string>
+    <string name="pref_nospam">NoSpam</string>
+    <string name="pref_nospam_description">Changing your NoSpam alters your Tox ID, meaning friend requests must be sent to the updated one.</string>
```
Comment (robinlinden) [inline atox/src/main/res/values/strings.xml:149]: Please drop the full stop here too. :P

**Review by robinlinden — APPROVED** [review]
This looks great, thank you! Sorry about forcing you over to GitHub from Weblate. :) Editing the English version over there led to very annoying conflicts and to the Weblate continously getting frozen. :P

Just the two minor punctuation nitpicks.

**Comment by robinlinden** [comment]
Oh, I see you made the changes in the GitHub webgui. If you're fine with it, I can make the changes locally and push to your branch?

File: atox/src/main/res/values/strings.xml (hunk 3 above)
Comment (comradekingu) [inline atox/src/main/res/values/strings.xml:125]:
```suggestion
    <string name="warn_proxy_broken">Set up your proxy correctly (or turn it off) before using the app</string>
```

File: atox/src/main/res/values/strings.xml (hunk 4 above)
Comment (comradekingu) [inline atox/src/main/res/values/strings.xml:149]:
```suggestion
    <string name="pref_nospam_description">Changing your NoSpam alters your Tox ID, meaning friend requests must be sent to the updated one</string>
```

**Comment by comradekingu** [comment]
@robinlinden Works now? Unless you want me to squash?

File: atox/src/main/res/values/strings.xml (hunk 3 above)
Comment (robinlinden) [reply atox/src/main/res/values/strings.xml]: If you want, we can talk about how the text is set up in the app, but that might be better suited to IRC or Tox than GitHub comments? :P Right now, the only text that has any full stops at the end are the ones that are multiple sentences.

**Pull request merged**